thrusterCalibration/runThrusters.ino

```diff
@@ -93,34 +135,10 @@ void setupRelay() {
   delay(2000);
 }
 
-void setup() {
-  Serial.begin(9600);
-
-  pinMode(starterPin, INPUT);
-
-  while( digitalRead(starterPin) != HIGH ) {
-    Serial.println("waiting");
-  }
-  Serial.println("starting..");
-  killSwitch = 1;
-  
-  delay(5000);
-
-  setupServo(servo2, 2);
-  setupServo(servo3, 3);
-  setupServo(servo4, 4);
-  setupServo(servo5, 5);
-//  setupServo(servo6, 6);
-//  setupServo(servo7, 7);
-  Serial3.begin(9600);
-  Serial.begin(9600);
-  delay(2000);
-  
-}
-
 void myLoop() {
 
   setSerial3Buffer(); // set buffer from user input
+  //What does 48 represent?
```

Comment:
The -48 was to pull integers out of the buffer, because this supports user-provided input.